protos/service_mesh_performance.proto

```diff
@@ -166,4 +175,5 @@ message EnvironmentConfig {
   string kubernetes_version = 4;
   repeated Node node = 5;
```

Comment:
As node is repeated, it should be nodes.
Same for endpoint_url ( line 35, couldn't comment on that )

Reply:
There are several repeated valus like smp version and service mesh version which signifies different environments in different categories.

Reply:
fixed